Code/Framework/AzCore/AzCore/IO/Streamer/Scheduler.h

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include <AzCore/IO/Streamer/FileRequest.h>
 #include <AzCore/IO/IStreamerTypes.h>
```

Comment:
It looks like the need for this include is only for accessing the `FileRequestPtr` type alias.
In theory this file could make another type alias for it to avoid the forward declaration
i.e
```
namespace AZ::IO
{
    class ExternalFileRequest;
    inline namespace SchedularInternal
    {
        using ExternalRequestPtr = AZStd::intrusive_ptr<ExternalFileRequest>;
    }
}
```